expo/notifications.go

```diff
@@ -0,0 +1,193 @@
+package expo
+
+// Adapted from https://github.com/oliveroneill/exponent-server-sdk-golang
```

Comment:
The only real changes were saving the receipt ID & using the Expo Push Token to authorize the request. Maybe it would be better to just fork it and make those changes.

Reply:
I'm going to just import a branch of this instead.